plugin/pkg/scheduler/factory/plugins.go

```diff
@@ -46,20 +60,26 @@ type AlgorithmProviderConfig struct {
 	PriorityFunctionKeys util.StringSet
 }
 
-// Registers a fit predicate with the algorithm registry. Returns the name,
-// with which the predicate was registered.
+// RegisterFitPredicate registers a fit predicate with the algorithm
```

Comment:
Its a godoc convention. From the blog, a godoc "comment is a complete sentence that begins with the name of the element it describes." govet issues a warning if you don't do this I believe.